templates/repo/home.tmpl

```diff
@@ -48,7 +48,7 @@
                     {{ $l := Subtract $n 1}}
                     {{range $i, $v := .Treenames}}
                         {{if eq $i $l}}
-                        <span class="bread">{{EscapePound $v}}</span>
+                        <span class="bread">{{$v}}</span>
```

Comment:
Yes, instead of `fcfg-eth1%3A1` it was showing filename as `ifcfg-eth1%253A2` i.e. double escaped. Same would happen for filenames with # I suppose, don't know what the original intent was.